geometry_msgs/msg/VelocityStamped.msg

```diff
@@ -0,0 +1,11 @@
+# This expresses the timestamped velocity of a frame 'body_frame_id' in the reference frame 'header.frame_id' expressed from arbitrary observation pose 'observation_pose'.
+# The message is often used in simplified versions where:
```

Comment:
```suggestion
```

With the removal of the PoseStamped and only using the frame_ids this optional version is no longer necessary.